README.md

```diff
@@ -61,140 +61,44 @@ headers, body = event.to_http()
 requests.post("<some-url>", data=body, headers=headers)
 ```
 
-### Event base classes usage
- 
-Parsing upstream structured Event from HTTP request:
-
-```python
-import io
-
-from cloudevents.sdk.event import v1
-from cloudevents.sdk import marshaller
-
-m = marshaller.NewDefaultHTTPMarshaller()
-
-event = m.FromRequest(
-    v1.Event(),
-    {"content-type": "application/cloudevents+json"},
-    """
-    {
-        "specversion": "1.0",
-        "datacontenttype": "application/json",
-        "type": "word.found.name",
-        "id": "96fb5f0b-001e-0108-6dfe-da6e2806f124",
-        "time": "2018-10-23T12:28:22.4579346Z",
-        "source": "<source-url>"
-    }
-    """,
-)
-```
-
-Parsing upstream binary Event from HTTP request:
+You can find a complete example of turning a CloudEvent into a HTTP request [here](samples/http-cloudevents/client.py).
 
-```python
-import io
-
-from cloudevents.sdk.event import v1
-from cloudevents.sdk import marshaller
-
-m = marshaller.NewDefaultHTTPMarshaller()
-
-event = m.FromRequest(
-    v1.Event(),
-    {
-        "ce-specversion": "1.0",
-        "content-type": "application/octet-stream",
-        "ce-type": "word.found.name",
-        "ce-id": "96fb5f0b-001e-0108-6dfe-da6e2806f124",
-        "ce-time": "2018-10-23T12:28:22.4579346Z",
-        "ce-source": "<source-url>",
-    },
-    b"this is where your CloudEvent data is, including image/jpeg",
-    lambda x: x,  # Do not decode body as JSON
-)
-```
+#### Request to CloudEvent
 
-Creating HTTP request from CloudEvent:
+The code below shows how to consume a cloudevent using the popular python web framework
+[flask](https://flask.palletsprojects.com/en/1.1.x/quickstart/):
 
 ```python
-from cloudevents.sdk import converters
-from cloudevents.sdk import marshaller
-from cloudevents.sdk.converters import structured
-from cloudevents.sdk.event import v1
-
-event = (
-    v1.Event()
-    .SetContentType("application/json")
-    .SetData('{"name":"john"}')
-    .SetEventID("my-id")
-    .SetSource("from-galaxy-far-far-away")
-    .SetEventTime("tomorrow")
```

Comment:
Is this API for event building no longer used/recommended? I see it's been removed from the samples.

Reply:
Yes, it's not very pythonic, and doesn't align well with the 1.0 spec.